build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -185,14 +201,29 @@ class RuntimeTestConfig {
     this.client.mocha.grep = !!argv.grep;
     this.client.verboseLogging = !!argv.verbose || !!argv.v;
     this.client.captureConsole = !!argv.verbose || !!argv.v || !!argv.files;
-    this.browserify.persistentCache = getPersistentBrowserifyCache();
-    this.browserify.configure = function (bundle) {
-      bundle.on('prebundle', function () {
-        log(
-          green('Transforming tests with'),
-          cyan('browserify') + green('...')
-        );
-      });
+
+    const importPathPlugin = {
+      name: 'import-path',
+      setup(build) {
+        build.onResolve({filter: /^[\w-]+$/}, (file) => {
+          if (file.path === 'stream') {
+            return {path: require.resolve('stream-browserify')};
```

Comment:
According to the [stream-browserify](https://github.com/browserify/stream-browserify) docs, you should be able to use native nodejs's `stream` module as a direct replacement.

```suggestion
            return {path: require.resolve('stream')};
```

FMI: can you explain what the `import-path` plugin does / why we need it?

Reply:
The docs are written as if you're using `browserify`, which we are not. Browserify will automatically remap imports of `stream` to `stream-browserify`, which is how the browser is supported. Since we aren't using Browserify, we need to do the mapping ourselves.